identity/terraform/provider.tf

```diff
@@ -1,24 +1,48 @@
+locals {
+  default_tags = {
+    TerraformConfigurationURL = "https://github.com/wellcomecollection/wellcomecollection.org/tree/main/identity"
+    Department                = "Digital Platform"
+    Division                  = "Culture and Society"
+    Use                       = "Identity APIs"
```

Comment:
What is `Use`?

Reply:
It describes how we're using these resources – it's for within our team.

Other examples: `CALM adapter`, `Goobi`, `Archivematica`.

Reply:
Sorry, I was reading it as you-z rather than you-s